### ISSUE TITLE
Worms 3D can freeze while in game

### ISSUE THREAD

**Issue by Mimimax** [FIX94/Nintendont#988]
Not sure if this applies to every game mode, but at least while in the 1P tutorial in any of its missions. Whenever a Worm is about to explode after being at 0 health or a ingame cutscene is about to start, the game has a chance of freezing, making you lose the entire progress of the mode no matter in which mission you are.

Tested on a normal Wii with GC controller, real memory card, USB Loader GX in a 4gb SDHC card and the game in a Samsung 300gb portable HDD.

**Comment by carnage702** [comment]
take a picture of your nintendont settings.

**Comment by carnage702** [comment]
is your iso a full 1.35gb iso with the md5 matching redump website?
you can check your iso md5 using this program

https://www.mediafire.com/file/5y3tfpoil433rij/MD5_SHA-1_Utility.exe/file

**Comment by Mimimax** [comment]
Thanks, checked. Full size and fully matches redump (USA version)

**Comment by carnage702** [comment]
well i have three suggestions

1- turn unlock read speed to on and try playing the game
2-try using an emulated memory card instead of real one, since even real ones of 1019 blocks sometimes have issues altough i doubt its the case here tbh

3- try putting the iso on the sd card and playing the game trough the sd card, to see if it still crashes.

**Comment by Mimimax** [comment]
I did the latter two suggestions together (forgot the first) and didn't get any crash at all this time. Then I tried the iso via the HDD again but with the emulated memory card still, and nothing had caused a crash either. So it's probably a 1019 card issue after all.
 
https://www.nintendo.com/consumer/memorycard1019.jsp

While Nintendo has this web page that lists some known games and their issues with this card, Worms 3D isn't present there, so I had no idea about this before. Also all of those games are 3rd party titles, and most of the games I've played via Nintendont with this card are 1st party or nintendo ip titles by 3rd parties, which I've had no problems with.
